src/views/Pages/Login/Login.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+// import { Offline, Online } from "react-detect-offline";
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { fetchUserData } from '../../../store/actions/user'
+import { isAuthEmail, authenticateUser, isAuthUserType } from '../../../services/Auth'
+import { Button, Card, CardBody, CardGroup, Col, Container, Form, Input, InputGroup, InputGroupAddon, InputGroupText, Row } from 'reactstrap';
+
+
+class Login extends Component {
+  constructor(props) {
+		super(props)
+		this.state = {
+			email_signIn: '',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces but found 3 tabs.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)